project/BuildHelper.scala

```diff
@@ -84,5 +88,6 @@ object BuildHelper extends ScalaSettings {
     Test / parallelExecution := true,
     incOptions ~= (_.withLogRecompileOnMacro(false)),
     autoAPIMappings := true,
+    javaOptions ++= javaxOptions
```

Comment:
These settings are only used by the `example` project. This way it's being added globally.